hazelcast/include/hazelcast/util/empty_function.h

```diff
@@ -21,10 +21,11 @@
 namespace hazelcast {
     namespace util {
         /**
-        * Generic empty body event handler
-        * \tparam EventT event type
+        * Generic empty body function
+        * \tparam R return type
+        * \tp
         */
-        template<typename EventT>
-        HAZELCAST_API void nullEventHandler(const EventT &) {}
+        template<typename R, typename ...Args>
+        HAZELCAST_API R empty_function(Args...) {}
```

Comment:
naming suggestion: empty-> noop

Reply:
I wasn't happy with that name either, noop is better. I also realized templating on the return type (R) didn't make much sense and changed it to be always `void`.